src/CommunityToolkit.Aspire.Hosting.Solr/CommunityToolkit.Aspire.Hosting.Solr.csproj

```
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <Description>A .NET Aspire hosting integration for Apache Solr.</Description>
+    <PackageTags>aspire solr search hosting</PackageTags>
```

```suggestion
    <PackageTags>solr search hosting</PackageTags>
```

We'll append `aspire` automatically